tpm/aziot-tpmd-config/src/lib.rs

```diff
@@ -15,6 +36,41 @@ pub struct Config {
     pub endpoints: Endpoints,
 }
 
+/*
+#[derive(Debug, Deserialize, PartialEq, Serialize)]
+pub struct TpmKeyConfig {
+    #[serde(deserialize_with = "persistent_index")]
+    pub index: u32,
+    pub overwrite: bool,
+}
+*/
+
+#[derive(Debug, Default, Deserialize, PartialEq, Serialize)]
```

Comment:
Do we want this to be configurable through the super-config? If so, also need to update `aziotctl/config/unix/template.toml` and implement it in the aziotctl-config-apply code.

Reply:
Yes, this should be configurable through the super-config.

Reply:
ba9741cf225d05e928f2005be9272f30d56238dc
88b47e884d452a43b56a95a2aab98a989472dca7